indexer/packages/postgres/src/stores/subaccount-table.ts

```diff
@@ -68,7 +67,7 @@ export async function findAll(
     baseQuery = baseQuery.where(SubaccountColumns.address, address);
   }
 
-  if (subaccountNumber) {
+  if (subaccountNumber !== undefined) {
```

Comment:
Bug with how subaccount =0 is handled. Will make a broader PR for all number based conditional where statements in another PR. https://linear.app/dydx/issue/OTE-576/update-ts-tables-where-conditionals-to-fix-queries-with-numbers